specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json

```diff
@@ -2127,6 +2416,122 @@
           "type": "string"
         }
       }
+    },
+    "EnterpriseChannelCheckNameAvailabilityRequestBody": {
+      "description": "The request body for a request to Bot Service Management to check availability of an Enterprise Channel name.",
```

Comment:
Please avoid REST API terminology in the parameter descriptions:
e.g. "A request to Bot Service Management to check availability of....."